API/Services/Tasks/VersionUpdaterService.cs

```diff
@@ -90,12 +90,18 @@ public async Task<IEnumerable<UpdateNotificationDto>> GetAllReleases()
         private UpdateNotificationDto CreateDto(GithubReleaseMetadata update)
         {
             if (update == null || string.IsNullOrEmpty(update.Tag_Name)) return null;
-            var version = update.Tag_Name.Replace("v", string.Empty);
-            var updateVersion = new Version(version);
+            var updateVersion = new Version(update.Tag_Name.Replace("v", string.Empty));
+            var currentVersion = BuildInfo.Version.ToString();
+
+            if (updateVersion.Revision == -1)
+            {
+                currentVersion = BuildInfo.Version.ToString()
```

Comment:
cache this into a variable to avoid duplicate string creations

Reply:
So you mean instead of modifying the currentVersion variable, create a new variable?

Reply:
No, I mean just cache the BuildInfo.Version.ToString() then use that in the following lines of code. 

Essentially, just want to avoid 2 string creations.

Reply:
Ah, I understand now. Got it!